aws-auth-cm.yaml

```diff
@@ -5,8 +5,12 @@ metadata:
   namespace: kube-system
 data:
   mapRoles: |
-    - rolearn: arn:aws:iam::014669633018:role/takeon-dev-stack-NodeInstanceRole-VCQY83J1O8L4
+    - rolearn: arn:aws:iam::014669633018:role/node-stack-NodeInstanceRole-1GCJ8ETD6WEDK
```

Comment:
Something we still need to automate?

Reply:
Yeah, I think this can be done by using local exec and running the aws command

Reply:
I will raise a tech debt for this